SIBR.Storage.Data/Models/EntityUpdateResponse.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Text.Json;
+using NodaTime;
+using SIBR.Storage.Data.Query;
+
+namespace SIBR.Storage.Data.Models
+{
+    public class EntityUpdateResponse
+    {
+        public UpdateType Type { get; set; }
+        public Guid EntityId { get; set; }
+        public Guid Hash { get; set; }
+        public Instant Timestamp { get; set; }
+        public JsonElement Data { get; set; }
```

Comment:
do you *need* to return the actual object itself? there's going to be very long spans of the same update with the same hash (tens/hundreds of thousands potentially) and it'll be very redundant. maybe returning the hash is enough?

Reply:
I don't need to, but I expect chron will get a rewrite for soon and nobody else will want this until then, and I have everything cached on disk by URL anyway so I'll only hit it once. It didn't seem worth adding the extra complexity. I could probably get by with just the hash if you're concerned about traffic.

Reply:
i don't think i mind particularly? it wouldn't be extra complexity to remove it (just remove the fields from the classes), but if you're cool with the extra data load then keeping it works too

Reply:
I would be extra complexity for me to then go get the object that the hash refers to. If there was already an endpoint to get objects by hash that would be one thing, but I had enough trouble making this one. Feel free to redesign it if you do a rewrite.

Reply:
ok with leaving it like this then